example/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "@craftzdog/react-native-buffer": "^6.0.5",
+    "bob": "^2.7.0",
```

Comment:
This looks incorrect

Reply:
This was accidentally commited - sorry. I removed it.